public/i18n/en.json

```diff
@@ -17,5 +17,6 @@
   "query-builder-add-condition": "Add condition",
   "query-builder-value-type-relation-dropdown-matching": "matching",
   "query-builder-value-type-relation-dropdown-without": "without",
-  "query-builder-value-type-relation-dropdown-regardless-of-value": "regardless of value"
+  "query-builder-value-type-relation-dropdown-regardless-of-value": "regardless of value",
+  "query-builder-limit-number-results-description": "Limit the number of results to"
```

Comment:
I think the copy doesn't really belong in this PR. Mainly, because I'm not sure that it is already final and so we shouldn't make translators work on things that might be outdated by tomorrow. Let's include that copy as literal text in the component for now

Reply:
It is not clear in the tickets that this is not the final copy. There is an older version somewhere but this is final as far as I am concerned. Did I miss some discussion about this?

Reply:
We can discuss it tomorrow in the meeting and leave it here for now maybe

Reply:
It has been decided that this is the final text for now.